tests/hrex/test_hrex_rbfe.py

```diff
@@ -114,6 +114,9 @@ def sample_and_record_rss(*args, **kwargs):
     final_replica_state_counts = result.hrex_diagnostics.cumulative_replica_state_counts[-1]
     assert np.any(np.all(final_replica_state_counts > 0, axis=0))
 
+    assert isinstance(result.hrex_diagnostics.relaxation_time, float)
```

Comment:
Noticed we don't actually call this code anywhere, not sure what a good test for it would be, but thought at least verifying that it doesn't trigger an exception would be good.